lib/ansible/plugins/filter/to_json.yml

```
@@ -0,0 +1,84 @@
+DOCUMENTATION:
+  name: to_json
+  author: core team
+  version_added: 'historical'
+  short_description: Convert variable to JSON string
+  description:
+    - Converts an Ansible variable into a JSON string representation. This filter functions as a wrapper to the Python ``json.dumps()`` function.
+    - Ansible internally auto-converts JSON strings into variable structures so this plugin is used to force it into a JSON string.
+  options:
+    _input:
+      description: A variable or expression that returns a data structure
+      type: raw
+      required: true
+      format: positional
+    vault_to_text:
+      description: Toggle to either unvault a vault or create the JSON version of a vaulted object
+      type: bool
+      default: True
+      format: named
+      version_added: '2.9'
+    preprocess_unsafe:
+      description: Toggle to represent unsafe values directly in JSON or create a unsafe object in JSON
+      type: bool
+      default: True
+      format: named
+      version_added: '2.9'
+    allow_nan:
+      description: When off, strict adherence to float value limits of the JSON spec, so ``nan``, ``-inf`` and ``-inf`` values will produce errors
+                   If on, JavaScript equivalents will be used (``NaN``, ``Infinity``, ``-Infinity``)
```

One of the `-`s has to be removed.
```suggestion
      description: When off, strict adherence to float value limits of the JSON spec, so C(nan), C(inf) and C(-inf) values will produce errors
                   If on, JavaScript equivalents will be used (C(NaN), C(Infinity), C(-Infinity))
```